keps/sig-apps/3017-pod-healthy-policy-for-pdb/README.md

```diff
@@ -530,6 +532,7 @@ For each of them, fill in the following information by copying the below templat
 - 2021-10-24: Proposed KEP for adding the new behavior in alpha status in 1.24.
 - 2022-11-11: Initial alpha implementation merged into 1.26
 - 2022-12-07: KEP rewritten to match the implementation (PodHealthyPolicy was renamed to UnhealthyPodEvictionPolicy)
+- 2023-01-24: Update for beta promotion
```

Comment:
Also, https://github.com/kubernetes/enhancements/blob/458d4689ad80b05ad938e97c5c29f354acf3df66/keps/sig-apps/3017-pod-healthy-policy-for-pdb/README.md#what-are-the-slis-service-level-indicators-an-operator-can-use-to-determine-the-health-of-the-service is not filled out. Neither is https://github.com/kubernetes/enhancements/blob/458d4689ad80b05ad938e97c5c29f354acf3df66/keps/sig-apps/3017-pod-healthy-policy-for-pdb/README.md#how-can-someone-using-this-feature-know-that-it-is-working-for-their-instance.